src/views/MorphicBarEditor.vue

```diff
@@ -888,8 +883,8 @@ export default {
         return false
       }
     },
-    editSubKindIcons: function() {
-      let data = {}
+    editSubKindIcons: function () {
+      const data = {}
```

Comment:
Same here, empty constant that you are starting modifying right away, makes no sense.